src/base/iconprovider.h

```diff
@@ -43,7 +44,7 @@ class IconProvider : public QObject
     static void freeInstance();
     static IconProvider *instance();
 
-    virtual QString getIconPath(const QString &iconId);
+    virtual QString getIconPath(const QString &iconId, const QString &overrideTheme = QString());
```

Comment:
I wasn't intending to change the file layout at all, but there is already a `src/icons/flags/` directory, so there's precedence for doing this.

@sledgehammer999 Do you want me to reorganise so there's a `src/icons/status/` directory?